Source/Core/DolphinWX/Debugger/MemoryWindow.cpp

```diff
@@ -94,7 +94,13 @@ CMemoryWindow::CMemoryWindow(wxWindow* parent, wxWindowID id,
 	sizerBig->Add(sizerRight, 0, wxEXPAND | wxALL, 3);
 	sizerRight->Add(addrbox = new wxTextCtrl(this, IDM_MEM_ADDRBOX, ""));
 	sizerRight->Add(valbox = new wxTextCtrl(this, IDM_VALBOX, ""));
-	sizerRight->Add(new wxButton(this, IDM_SETVALBUTTON, _("Set &Value")));
+	sizerRight->Add(new wxButton(this, IDM_SETVALBUTTON,
+#ifdef __APPLE__
+		_("Set Value")
+#else
+		_("Set &Value")
+#endif
```

Comment:
Okay.  There are also a bunch of other dubious accelerators - two options labeled `&D`, three `&U`.  I guess I should just remove all the accelerators in the file (reminder to self: see whether pressing enter on the value textbox triggers Set Value).